tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -269,10 +269,9 @@ sub prepare_guest_for_sriov_passthrough {
     }
 
     # Journals with previous reboot
-    my $journald_conf_file = "/etc/systemd/journald.conf";
-    if (!script_run "ssh root\@$vm \"ls $journald_conf_file\"") {
+    if (guest_is_sle($vm, '<=15-sp5')) {
```

Comment:
If there is specific handling for guests<=15sp5, why not add some branch to do handling for guests>=15sp6? It will save those repeating lines in several guest unattended installation files. It seems better to me. Do I overlook anything that convince you pick the other way?

Reply:
If you are asking why I configure journald.conf for sle15sp5- only but not for sle15sp6+, it is because that the configuration for sle15sp6+ is different and made in guest autoyast profiles.

Reply:
The advantage of configuring journald in guest autoyast profile is that we can get entire guest log at the beginning of all feature tests, not limited to sriov tests. And making change in autoyast profile is faster and more stable than in 'ssh \@guest_vm xxxx'.

Reply:
You got my point in the second reply :)
If we need entire guest log from the very beginning, setting in autoyast is a good way. Then 
- why don't we set it in  autoyast for all guests(differently for sle15sp6 and earlier versions)? 
- with autoyast set for journal config, we can save the modifications and restart in sriov?

Reply:
> You got my point in the second reply :) If we need entire guest log from the very beginning, setting in autoyast is a good way. Then
> 
> * why don't we set it in  autoyast for all guests(differently for sle15sp6 and earlier versions)?
two reasons:

1. earlier versions mainly include sle15sp5 and sle12sp5. For UEFI guests, we use autoyast profiles rather than autoyast template of host installation, it does not support something like "if/else" macros, ie. we have to add seperate autoyast profiles for sle15sp5(I guess 4 more files); For legacy guests, we need modify sle15sp5 fv/pv and sle12sp5 fv/pv autoyast profiles too. Too much changes.
2. since current logic can work partly for earlier versions, and sle15sp5 will be obsolote soon, I think this change would cost less but gain more. 

> * with autoyast set for journal config, we can save the modifications and restart in sriov?

"we can save the modifications and restart in sriov?" not really understand this question well. the journald will be well ready (with reboot logs) after autoyast installation, what modification or restart are needed in sriov?

Reply:
We discussed this offline. Agreement is that we will create a new ticket to write a new module for updating guests after guest installation and before feature tests, so that future similar changes can be added in that new module, rather than updating so many guest autoyast files. So will merge this as it is now. Julie will redesign it later in that new created ticket.

Reply:
We discussed this offline. Agreement is that we will create a new ticket to write a new module for updating guests after guest installation and before feature tests, so that future similar changes can be added in that new module, rather than updating so many guest autoyast files. So will merge this as it is now. Julie will redesign it later in that new created ticket.